src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-druid-integration.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Apache Druid integration
+tags:
+  - Druid integration
+  - apache druid integration
+  - New Relic integrations
+metaDescription: Use New Relic infrastructure agent to get a dashboard with druid metrics.
+freshnessValidatedDate: 2024-04-21
+---
+
+import infrastructureDruidDashboard from 'images/infrastructure_screenshot-full_apache-druid-dashboard.webp'
+
+Integrating Apache Druid with New Relic enhances your ability to monitor, analyze, and optimize the performance of your Druid clusters. By leveraging New Relic's powerful monitoring and alerting features, you can ensure the reliability and efficiency of your Druid-based analytics platform.
+
+<img
+    title="Apache Druid dashboard"
+    alt="Apache Druid dashboard"
+    src={infrastructureDruidDashboard}
+/>
+
+## Set up the Apache Druid integration
+
+Complete the following steps to set up the Apache Druid integration:
+
+<Steps>
+    <Step>
+## Install the infrastructure agent
+To use the Apache Druid integration, you need to also [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. The infrastructure agent monitors the host itself, while the integration you'll install in the next step extends your monitoring with Apache Druid-specific data.    
+    </Step>
+    <Step>
+## Expose Druid metrics using prometheus emitter [#Expose]
+
+1. Add `prometheus.emitter` to the end of the extensions load list in your `apache-druid-24.0.0/conf/druid/single-server/micro-quickstart/_common/common.runtime.properties` file:
+
+```yml
+druid.extensions.loadList=["druid-hdfs-storage", "druid-kafka-indexing-service", "druid-datasketches", "druid-multi-stage-query", "prometheus-emitter"]
+```
+
+2. Add the code snippets in the right column to the files in the left column:
```

Comment:
I would make this sentence parallel left --> right of the table, so either add the snippets to the left column and file paths to the right (Add the code snippets in the left column to the file paths in the right column), or change the wording to: 

"In the file paths listed in the left column, add the code snippets listed in the right columns."

Reply:
i know this is a huge pain but the parallelism will def help